Ginger/GingerCoreNET/Drivers/CoreDrivers/Mobile/Appium/GenericAppiumDriver.cs

```diff
@@ -4278,5 +4732,562 @@ public void InstallApp(string appPath)
             }
 
         }
+        public void EmulateIncomingCall(string phoneNumber)
```

Comment:
remove methods which not working and not used in mobile action eventually to avoid confusion